main.go

```diff
@@ -8,6 +8,11 @@ import (
 	log "github.com/Sirupsen/logrus"
 	"github.com/rancherio/go-machine-service/events"
 	"github.com/rancherio/go-machine-service/handlers"
+	_ "github.com/rancherio/go-machine-service/handlers/providers/amazonec2"
```

Comment:
if you look at azure.go, it has moved from handlers/providers/azure.go → handlers/providers/azure/azure.go